crates/ruff/src/rules/flake8_pyi/mod.rs

```diff
@@ -0,0 +1,26 @@
+//! Rules from [flake8-pyi](https://pypi.org/project/flake8-pyi/).
+pub(crate) mod rules;
+
+#[cfg(test)]
+mod tests {
+    use std::path::Path;
+
+    use anyhow::Result;
+    use test_case::test_case;
+
+    use crate::registry::Rule;
+    use crate::test::test_path;
+    use crate::{assert_yaml_snapshot, settings};
+
+    // TODO: figure out how to have another test case for Y001 that is a .py file.
```

Comment:
I wasn't sure how to have multiple test files, in this case I want to make sure we ignore `.py` files since they don't have the same import/export rules -- although maybe it should apply to all files anyways? not sure when you'd want to export a type var

Reply:
Grep for `#[test_case(Rule::InvalidEscapeSequence, Path::new("W605_0.py"))]` as an example -- that's our standard convention.